x-pack/plugins/security_solution/public/cases/components/add_comment/index.tsx

```diff
@@ -62,6 +71,28 @@ export const AddComment = React.memo<AddCommentProps>(
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [insertQuote]);
 
+    const handleTimelineClick = useCallback(
```

Comment:
this is an unrelated bug fix, i didn't do the hijack click in the add comment preview. whoops! if reviewer wants this in a separate PR, I can do that. needed it to make a pretty gif :)